lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -308,8 +318,12 @@ def create_resource_condition(record: dict, symptom_name: str, patient_reference
         "subject": patient_reference
     }
 
-    if record["symptom_duration"]:
-        condition["onsetDateTime"] = record["symptom_duration"]
+    symptom_duration = record.get(f'symptom_duration{suffix}')
+
+    # Defend against known data quality issue in REDCap where symptom duration
+    # dates can come from the future.
+    if symptom_duration and datetime.strptime(symptom_duration, '%Y-%m-%d') < datetime.now():
+        condition["onsetDateTime"] = symptom_duration
```

Comment:
on second thought, this may be better as a separate commit, since it will affect `symptoms` and `symptom_duration` (not just `*_2`).

happy to split it out if there's a preference

Reply:
we're no longer doing this.